api/src/services/setup/upgrade-steps.js

```diff
@@ -116,7 +116,20 @@ const complete = async (buildInfo) => {
   const stagingDoc = await upgradeUtils.getStagingDoc(buildInfo);
   const payload = upgradeUtils.getUpgradeServicePayload(stagingDoc);
 
-  return await upgradeUtils.makeUpgradeRequest(payload);
+  const response = await upgradeUtils.makeUpgradeRequest(payload);
+
```

Comment:
If successful then API will be restarted before execution makes it past this line, right? It's probably worth a comment or similar in the code, because it's not an obvious code exit point...

Reply:
> If successful then API will be restarted before execution makes it past this line

This is correct. Adding the comment.